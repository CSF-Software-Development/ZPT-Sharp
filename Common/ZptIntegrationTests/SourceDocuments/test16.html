<a href="valid/link.html"
   tal:define="origHref attrs/href"
   tal:attributes="href string:/base/$origHref">blah, blah</a>
