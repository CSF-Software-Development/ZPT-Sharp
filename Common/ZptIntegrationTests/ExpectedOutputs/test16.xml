<?xml version="1.0" encoding="utf-8"?>
<body>

<ImG href="foo" Alt="bar" Href="about:foo" alT="baz" />

</body>